trackintel/preprocessing/positionfixes.py

```diff
@@ -85,6 +91,17 @@ def generate_staypoints(
     # copy the original pfs for adding 'staypoint_id' column
     pfs = pfs_input.copy()
 
+    if exclude_duplicate_pfs:
```

Comment:
I think this idea is nice. I am just thinking since the records are totally the same, why would the user want to keep them? Deleting them might be the default and always used option? 

I feel like we have a lot of input parameters for this function and add one extra `exclude_duplicate_pfs` seems unnecessary in most use cases

Reply:
I agree with dropping duplicates being the default option but I don't see how an additional keyword would hurt the function. In general I think that more customization is better as long as the default value and the parameter explanation are good.